### PR TITLE
Add Dependabot

### PR DESCRIPTION
**Pull request by seano-vs**
**What this does:**
Adds the boilerplate dependabot file we use for different repos.

_Side effects:_

New software deps could break something.

**Checklist:**

- [ ] This has been tested locally.
  - Notes:
- [ ] [Commented code](https://docs.google.com/document/d/1M_i2i3V2aNq6Yiofwj5Su8mKGLs1ooQOJGHR-37WZzs/edit) in a way that is useful for others.
  - Notes:
- [ ] Updated or created relevant documentation.
  - Notes:
- [ ] Added automated tests relevant to this new code.
  - Notes:

### CODE REVIEW

File: src/vape.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import schemathesis
 import os
 
-DEFAULT_TIMEOUT = 10
+DEFAULT_TIMEOUT = 1
```
Comment (zzolo) [inline src/vape.py:10]: This seems like an unrelated change?

**Review by zzolo — APPROVED** [review]
Noted something about the default timeout.  Assuming that was intentional, this is good.

File: src/vape.py (hunk 1 above)
Comment (seano-vs) [reply src/vape.py]: good catch! resolved
